src/test/java/io/dashboardhub/pipelinedashboard/definitions/ProjectDefinitions.java

```diff
@@ -3,10 +3,38 @@
 import cucumber.api.java.en.Then;
 import cucumber.api.java.en.When;
 import io.dashboardhub.pipelinedashboard.PipelinedashboardApplicationTests;
+import io.dashboardhub.pipelinedashboard.domain.Project;
+import io.dashboardhub.pipelinedashboard.domain.User;
+import io.dashboardhub.pipelinedashboard.repository.ProjectRepository;
+import io.dashboardhub.pipelinedashboard.repository.UserRepository;
 import org.junit.Assert;
+import org.springframework.beans.factory.annotation.Autowired;
 
 public class ProjectDefinitions extends PipelinedashboardApplicationTests {
```

Comment:
I'm confused here. I thought we were using cucumber purely for acceptance tests (full application tests via selenium). Are we now using the same for integration tests (which this class looks like)??